jaxrs-spec/pom.xml

```diff
@@ -22,7 +22,7 @@
 
     <groupId>jakarta.ws.rs</groupId>
     <artifactId>jakarta.ws.rs-spec</artifactId>
-    <version>2.1.6</version>
+    <version>2.1</version>
```

Comment:
I do not see who has a benefit from hiding the fact that this is just a bug fix of an existing specification.

Reply:
It is not a bug fix of an existing specification.  It's the first version of this specification approved by Eclipse.

Reply:
Maybe we should write 2.1-1 then? In maven words this means "second build of same version".